docs/INSTALLATION.md

```diff
@@ -0,0 +1,398 @@
+# Solutions Template - Installation
+
+Table of Content
+
+<!-- vscode-markdown-toc -->
+* 1. [Prerequisites](#Prerequisites)
+	* 1.1. [Understanding Google Cloud](#UnderstandingGoogleCloud)
+	* 1.2. [Tool requirements:](#Toolrequirements:)
```

Comment:
A few bullets not hyperlinked correctly in TOC

Reply:
Updated TOC